packages/lit-html/src/lit-html.ts

```diff
@@ -470,7 +474,8 @@ class Template {
 
     // Note, we don't add '</svg>' for SVG result types because the parser
     // will close the <svg> tag for us.
-    this._element.innerHTML = html + this._strings[l];
+    this._element = this._createElement(html + this._strings[l]);
```

Comment:
I'm interested in exploring removing classes for Template and TemplateInstance, since they're never really polymorphic. Even here, this is a global patch, so it's not polymorphic. I guess we can have a settable module variable, and we could possibly have a build config to replace it with a const if we want to make builds w/o IE support.

Reply:
Yup, seems possible.